terraform/modules/cloud/google/deploy/fleet/templates/user_data.bash

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Don't enable debug mode in this script if secrets are managed as it ends-up in the logs
+# set -x
+exec > >(tee /tmp/user-data.log|logger -t user-data ) 2>&1
+
+git clone -b ${bootstrap_version} --single-branch https://github.com/aeternity/infrastructure.git /infrastructure
+cd /infrastructure/ansible
+
+# Temporary fix/workaround for non-executable vault install
+chmod +x /usr/bin/vault
+
+# Authenticate the instance to CSM
+PKCS7=$(curl -s http://169.254.169.254/latest/dynamic/instance-identity/pkcs7 | tr -d '\n')
```

Comment:
I think this is not available in Google Cloud. They provide a [different mechanism](https://cloud.google.com/compute/docs/instances/verifying-instance-identity).

Reply:
yeap.. you are 100% right...